.azure-pipelines/docker-sonic-slave.yml

```diff
@@ -0,0 +1,102 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+# Build and push sonic-slave-[buster|jessie|stretch] images for amd64/armhf/arm64
+
+schedules:
+- cron: "0 8 * * *"
+  branches:
+    include:
+    - master
+  always: true
+
+trigger:
+  batch: false
+  branches:
+    include:
+    - master
```

Comment:
we do not want to trigger for every pr, only the pr that contains the changes to sonic-slave-* folder.